js/assessments/sentenceLengthInDescriptionAssessment.js

```diff
@@ -0,0 +1,58 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+var countTooLongSentences = require( "./checkForTooLongSentences.js" );
+
+/**
+ * Calculates sentence length score
+ * @param {array} sentences The array containing sentences.
+ * @param {object} i18n The object used for translations.
+ * @returns {object} Object containing score and text.
+ */
+var calculateSentenceLengthResult = function( sentences, i18n ) {
+	var recommendedValue = 20;
+	var tooLong = countTooLongSentences( sentences, recommendedValue );
+	var percentage = ( tooLong / sentences.length ) * 100;
+
+	// 6 is the number of scorepoints between 3, minscore and 9, maxscore. For scoring we use 10 steps
```

Comment:
This comment doesn't clarify the calculation under it at all. Maybe it is better to place this calculation on multiple lines and clarify each statement.